server.js

```diff
@@ -7,7 +7,9 @@ new WebpackDevServer(webpack(config), {
   contentBase: baseConfig.output.path,
   publicPath: baseConfig.output.publicPath,
   hot: process.env.NODE_ENV === 'development' ? true : false,
-  historyApiFallback: true,
+  historyApiFallback: {
+    disableDotRule: true,
+  },
```

Comment:
I should have been more specific: Can we just set historyApiFallback to false? Or even completely remove the property if its default value is false?
I guess we can still investigate that as a follow-up issue and land this now.